src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,37 @@
+<div class="footer">
```

Comment:
```suggestion
<footer class="footer">
```
セマンティクスタグを使えるところでは使っていきましょう！